app/main.py

```diff
@@ -12,7 +45,125 @@
     allow_headers=["*"],
 )
 
+class Request(BaseModel):
+    location: str
+    prompt:  str
+
+class Response(BaseModel):
+    id: int
+    name: str
+    long: float
+    lat: float
+    area: str
+    category: str
 
 @app.get("/")
 async def hello():
-    return {"message": "Hello World"}
\ No newline at end of file
+    return {"message": "Hello World"}
+
+@app.post("/api")
+async def search_pub(request: Request):
```

Comment:
アノテーションをつけましょう！今回はJSONを格納したリストをreturnするのでこんな感じかなー
```suggestion
async def search_pub(request: Request) -> List[Izakaya]:
```
ここで使用している`Izakaya`クラスは一つ一つの居酒屋情報(辞書)を表す型です。`app/models/izakaya.py`でモデル定義をしインポートして使ってみましょう。めっちゃついでなのですがpubという表現は適切じゃなさそうなのでpubが使用されているすべての部分をizakayaに変更しておいて欲しいです

Reply:
了解！！